train_gpt2.cu

```diff
@@ -634,7 +634,7 @@ __global__ void gelu_backward_kernel(floatX* dinp, const floatX* inp, const floa
 // the idea is to employ one block to reduce along several columns,
 // where each block has a width of 32 columns to ensure coalesced access.
 // at the end we accumulate the reductions performed by the warps in each block via shared memory
-__global__ void matmul_backward_bias_kernel4(floatX* dbias, const floatX* dout, int B, int T, int OC) {
+__global__ void matmul_backward_bias_kernel2(floatX* dbias, const floatX* dout, int B, int T, int OC) {
```

Comment:
should there be a number here?
not just `matmul_backward_bias_kernel`?

Reply:
the index of 2 is to be consistent of the index in the dev file (where the naive GPU kernel is indexed as 1).